alvr/client_openxr/src/interaction.rs

```diff
@@ -108,17 +108,17 @@ pub struct InteractionContext {
     pub action_set: xr::ActionSet,
     pub button_actions: HashMap<u64, ButtonAction>,
     pub hands_interaction: [HandInteraction; 2],
-    pub multimodal_hands_enabled: bool,
+    pub multimodal_hands: Option<(MultimodalMeta, bool)>,
```

Comment:
This grouping is awful to read in every place it gets used. You could put the bool into the Meta struct itself?
I actually think having a non pub `multimodal_hands` and a pub `multimodal_hands_enabled` would actually still be the best, because the cognitive complexity nesting the bool into the option produces is not counteracted by the small improvement it brings (that I'll explain in another comment)

Reply:
i'll keep the flag external. this was discussed before, we cannot create a stateful handle for SIMULTANEOUS_HANDS_AND_CONTROLLERS_EXTENSION because technically we could create a new second handle instance and that would mess things up